canvas_modules/common-canvas/__tests__/_utils_/property-utils.js

```diff
@@ -76,6 +76,51 @@ function flyoutEditorForm(paramDef, propertiesConfigOverrides, callbacksOverride
 	return { wrapper: wrapper, controller: renderedController, callbacks: callbacks };
 }
 
+function flyoutEditorFormRender(paramDef, propertiesConfigOverrides, callbacksOverrides, propertiesInfoOverrides) {
```

Comment:
Can you please define this in a way that we don't duplicate code ?

Reply:
`function flyoutEditorForm(paramDef, propertiesConfigOverrides, callbacksOverrides, propertiesInfoOverrides, isRender=false) {
	const applyPropertyChanges = sinon.spy();
	const closePropertiesDialog = sinon.spy();
	let callbacks = {
		applyPropertyChanges: applyPropertyChanges,
		closePropertiesDialog: closePropertiesDialog,
		controllerHandler: controllerHandler
	};
	if (callbacksOverrides) {
		callbacks = Object.assign(callbacks, callbacksOverrides);
	}

	let propertiesInfo = {
		parameterDef: cloneDeep(paramDef)
	};

	if (propertiesInfoOverrides) {
		propertiesInfo = Object.assign(propertiesInfo, propertiesInfoOverrides);
	}

	let propertiesConfig = {
		applyOnBlur: true,
		rightFlyout: true,
		trimSpaces: true,
		containerType: "Custom"
	};
	if (propertiesConfigOverrides) {
		propertiesConfig = Object.assign(propertiesConfig, propertiesConfigOverrides);
	}
	let wrapper;
	if (isRender){
		wrapper = renderWithIntl(
			<div className="properties-right-flyout">
				<CommonProperties
					propertiesInfo={propertiesInfo}
					propertiesConfig={propertiesConfig}
					callbacks={callbacks}
					customControls={[CustomTableControl, CustomToggleControl]}
					customConditionOps={[CustomOpMax]}
				/>
			</div>
		);
	}
	else {
		wrapper = mountWithIntl(
			<div className="properties-right-flyout">
				<CommonProperties
					propertiesInfo={propertiesInfo}
					propertiesConfig={propertiesConfig}
					callbacks={callbacks}
					customControls={[CustomTableControl, CustomToggleControl]}
					customConditionOps={[CustomOpMax]}
				/>
			</div>
		);
	}
	return { wrapper: wrapper, controller: renderedController, callbacks: callbacks };
}`

Reply:
How about adding an optional boolean to the arguments of flyoutEditorForm?
Lmk your thoughts on this and if you have any other suggestions

Reply:
That works too, but what will happen after enzyme is completely migrated do we need to update all functions whenever working on that particular test and after all functions are covered would need to remove that argument?

Reply:
What about this:

```
function flyoutEditorFormRender(a, b, c, d){
    const wrapper = flyoutEditorFrom(a, b, c, d, true);
    return wrapper;
}
```
as were migrating we call `flyoutEditorFormRender` and once it is fully migrated we can change the implementation of `flyoutEditorFormRender` to `flyoutEditorForm` but with RTL

Any thoughts on this implementation?

Reply:
Yes that works.

Reply:
I implemented my idea and tested it out and I'm getting the following
<img width="1237" alt="image" src="https://github.com/elyra-ai/canvas/assets/103902194/c76e67d7-996d-4707-a46e-fe535cc9a14d">

Reply:
Is this happening because you are reusing the function ? else you can split the function like below.

func a() {}
func b() {}

func mnt(){
a();
b();
return mount()
}

func rnder(){
a();
b();
return render()
}

Reply:
Hey I made another commit. Let me know what you think my new implementation.

Reply:
You can remove `flyoutEditorFormRender` if its not used in the code.